evalml/tests/data_checks_tests/test_highly_null_rows_data_check.py

```diff
@@ -0,0 +1,102 @@
+import numpy as np
+import pandas as pd
+import pytest
+import woodwork as ww
+
+from evalml.data_checks import (
+    DataCheckAction,
+    DataCheckActionCode,
+    DataCheckMessageCode,
+    DataCheckWarning,
+    HighlyNullRowsDataCheck
+)
+
+
+def test_highly_null_data_check_init():
+    highly_null_check = HighlyNullRowsDataCheck()
+    assert highly_null_check.pct_null_threshold == 0.5
+
+    highly_null_check = HighlyNullRowsDataCheck(pct_null_threshold=0.0)
+    assert highly_null_check.pct_null_threshold == 0
+
+    highly_null_check = HighlyNullRowsDataCheck(pct_null_threshold=0.5)
+    assert highly_null_check.pct_null_threshold == 0.5
+
+    highly_null_check = HighlyNullRowsDataCheck(pct_null_threshold=1.0)
+    assert highly_null_check.pct_null_threshold == 1.0
+
+    with pytest.raises(ValueError, match="pct_null_threshold must be a float between 0 and 1, inclusive."):
+        HighlyNullRowsDataCheck(pct_null_threshold=-0.1)
+    with pytest.raises(ValueError, match="pct_null_threshold must be a float between 0 and 1, inclusive."):
+        HighlyNullRowsDataCheck(pct_null_threshold=1.1)
+
+
+def test_highly_null_data_check_warnings():
+    data = pd.DataFrame({'a': [None, None, 10],
+                         'b': [None, "text", "text_1"]})
```

Comment:
Is None the only "null" we're considering here?  We might want to consider similar tests with np.nan or other equivalent types of "null".  I think you could probably parameterize it:
```
@pytest.mark.parametrize("null_type", [None, np.nan])
def test_highly_null_data_check_warnings(null_type):
    data = pd.DataFrame({'a': [null_type, null_type, 10],
                         'b': [null_type, "text", "text_1"]})
```
Or something like that.